fastlane/Fastfile

```diff
@@ -162,9 +163,27 @@ lane :release do
   # Puplish pod
   pod_push(path: "JOSESwift.podspec")
 
+  # Publish release
+  publish_release
+
   ensure_git_status_clean
 end
 
+desc "Publishes the release on GitHub"
+lane :publish_release do
+  version = get_version_number(target: "JOSESwift")
+
+  changelog = File.read("../CHANGELOG.md")
+  changes = changelog[/(- .+\(#.+\)\n)+/]
```

Comment:
What is this regex doing? I get you're stripping chars, but why exactly?

Reply:
Filtering out the latest change set.

``` 
Blablabla

[1.2.0]

- Do Something (#123)
- Do Something else (#100)

[1.2.0]

- Start something (#99)
```

Will become:

```
- Do Something (#123)
- Do Something else (#100)
```

Which is the change log for the latest version that we want attached to the release.

Reply:
Might be worth dropping a source comment

Reply:
Yeah you're right. 👍  Will add this later!